src/lib/Log.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * An utility class to print nice Log messages.
+ */
+export default class Log {
```

Comment:
I'm surprised there's not a library to do this. Maybe there's just not a nice and small library that does this.

Reply:
I kind of copied this from amp-toolbox: https://github.com/ampproject/amp-toolbox/blob/master/packages/core/lib/Log.js